phpstorm-url-handler

```diff
@@ -8,12 +8,12 @@
 # @author Stefan Auditor <stefan.auditor@erdfisch.de>
 
 arg=${1}
-pattern=".*file(:\/\/|\=)(.*)&line=(.*)"
+pattern=".*file(?::\/\/|\=)(.*)&line=(.*)"
```

Comment:
I tested Release 1.2.1 and it works perfectly! Thanks for making the changes to the master.